### ISSUE TITLE
Unique symbol property names are mangled

### ISSUE THREAD

**Issue by thebestnom** [TypeStrong/typedoc#1514]
## Search Terms

unique symbol, mangled name

## Problem

unique symbols looks weird in documentation because it get generated like a normal const

## Suggested Solution

use const name instead of generated const in `unique symbol`

```ts
export const NEOVIS_ADVANCED_CONFIG: unique symbol;
export interface Test {
  [NEOVIS_ADVANCED_CONFIG]?: string; // this will look bad in documentation
}
```
![image](https://user-images.githubusercontent.com/21955247/108610771-f9751300-73e0-11eb-8b25-22180e261783.png)

**Comment by Gerrit0** [comment]
This is a tricky one... related to #1226, though this is a particularly bad case. For now, I'll patch this with `[NEOVIZ_ADVANCED_CONFIG]` showing up as the property name... We really need to better handle dynamic property members...

**Comment by Gerrit0** [comment]
Fix released in 0.20.27
